src/component/tooltip/TooltipView.ts

```diff
@@ -171,8 +171,8 @@ class TooltipView extends ComponentView {
 
         this._tooltipContent = renderMode === 'richText'
             ? new TooltipRichContent(api)
-            : new TooltipHTMLContent(api.getDom(), api, {
-                appendToBody: tooltipModel.get('appendToBody', true)
+            : new TooltipHTMLContent(api, {
+		 appendTo: tooltipModel.get('appendToBody', true) ? 'body' : tooltipModel.get('appendTo', true),
```

Comment:
Sry. My vim does not read the prettier correctly. It will change the whole file to another format. So I have to use it with out any prettier plugin.

Will try to fix it this week.